Pseudo/css/pseudoClass.css

```diff
@@ -0,0 +1,57 @@
+h3 {
+	font-size: 20px;
+	line-height: 22px;
+	margin-bottom: 8px;
+}
+.section {
+	margin-bottom: 20px;
+}
+
+/*anchorStyling*/
+
+.anchorStyling a {
+	font-size: 16px;
+	line-height: 18px;
+}
+.anchorStyling a:link {
+	color: blue;
+}
+.anchorStyling a:visited {
+	color: red;
+}
+.anchorStyling a:hover {
+	color: green;
+}
+.anchorStyling a:active {
+	color: yellow;
+}
+
+/*Pseudo Child Classes*/
+
+.pseudoChild ul li a {
+	color: #ffffff;
+}
+.pseudoChild ul li {
+	margin-bottom: 10px;
+}
+.pseudoChild ul li:first-child {
+	background-color:green;
+}
+.pseudoChild ul li:last-child {
+	background-color: blue;
+}
+.pseudoChild ul li:nth-child(2),.pseudoChild ul li:nth-child(3) {
```

Comment:
It is understandable for this assignment that you are showing for demonstration purposes but generally we shouldn't use so many child selectors. Have the habit of using classes instead since the number of children might increase or decrease.

Reply:
ok